buildpack.md

```diff
@@ -891,9 +841,8 @@ value = "<label valu>"
 
 [[processes]]
 type = "<process type>"
-command = "<command>"
+command = ["<command>"]
 args = ["<arguments>"]
-direct = false
 default = false
```

Comment:
```suggestion
```

Reply:
Is there some more context for this? As far as I see, `default` sticks around in the RFC.